packages/devtools/README.md

```diff
@@ -1,78 +1,66 @@
+If you have `pub` on your path, you can run:
 
-## Development
+- `pub global activate devtools`
 
-- `git clone https://github.com/flutter/devtools`
-- `cd devtools/packages/devtools`
-- `pub get`
+If you have `flutter` on your path, you can run:
 
-From a separate terminal:
-- `cd <path/to/flutter-sdk>/examples/flutter_gallery`
-- ensure the iOS Simulator is open (or a physical device is connected)
-- `flutter run`
+- `flutter packages pub global activate devtools`
 
-From the packages/devtools directory:
-- `pub run webdev serve web`
+That will  install (or update) DevTools on your machine.
 
-Then, open a browser window to the local url specified by webdev. After the page has loaded, append
-`?port=xxx` to the url, where xxx is the port number of the service protocol port, as specified by 
-the `flutter run` output.
+Going forward, we expect to have additional (and easier!) distributions mechanisms for DevTools.
 
-For more productive development, launch your flutter application specifying
-`--observatory-port` so the observatory is available on a fixed port. This
-lets you avoid manually entering the observatory port parameter each time
-you launch the application.
+### Run the DevTools application server
 
-- `flutter run --observatory-port=8888`
-- `open http://localhost:8080/?port=8888`
+Next, you want to run the local web server which can serve the DevTools application itself.
 
-`pub run webdev` provides a fast development server that incrementally
-rebuilds the portion of the application that was edited each time you reload
-the page in the browser. If initial app load times become slow as this tool
-grows, we can integrate with the hot restart support in `webdev`.
+Run one of the following two commands; if you have `pub` on your path:
 
-## Testing
+- `pub global run devtools`
 
-### Running tests that depend on the Flutter SDK
-Make sure your flutter SDK matches the tip of trunk before
-running these tests.
+And if you have `flutter` on your path:
 
-```
-cd packages/devtools
-pub run test --tags useFlutterSdk
-```
+- `flutter packages pub global run devtools`
 
-### Run all other tests
+On the command-line, you should see output that looks something like:
 
 ```
-cd packages/devtools
-pub run test --exclude-tags useFlutterSdk
-pub run test --exclude-tags useFlutterSdk --platform chrome-no-sandbox
+Serving DevTools at http://127.0.0.1:9100
 ```
 
-### Updating golden files:
-Some of the golden file tests will fail if Flutter changes the implementation or diagnostic
-properties of widgets used by the inspector tests. If this happens, make sure the golden
-file output still looks reasonable and execute the following command to update the golden files.
+### Start an application to debug
+
+Next, you'll want to start an app to connect to. This can be either a Flutter application or a Dart
+command-line application. The example below uses a Flutter app.
+
+- change to the directory for a Flutter app
```

Comment:
maybe add `cd path/to/flutter/app` to be specific

Reply:
done